### PR TITLE
Internal site component - widget guidelines

### PR DESCRIPTION
**Pull request by tessarodes**
Added new component guidelines content for IBM Cloud widgets, including 4 tabs (style, usage, code and review) as well as associated images.

### CODE REVIEW

File: src/content/your-product-on-ibm-cloud/widget-guidelines/usage.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+tabs: ['Usage', 'Style', 'Code', 'Approval']
+internal: true
+---
+
+
+## General guidelines
+
+A <b>dashboard widget</b> provides a summary of a platform functionality, status, or individual service. The goal of a widget is to provide actionable information that helps the user quickly accomplish a key task or track key metrics. 
+
+<br>
```
Comment (alisonjoseph) [inline src/content/your-product-on-ibm-cloud/widget-guidelines/usage.mdx]:
```suggestion
<br />
```

needs to have a closing tag or be self closing or the build will fail

File: src/content/your-product-on-ibm-cloud/widget-guidelines/usage.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+### Widget size
+
+Dashboard widgets have <b>two possible sizes</b>, small and large. The height for both sizes is fixed (310px) and neither widget is currently resizable. 
+
+- Small (1:1) - Recommended 
+- Large (2:1) - Requires an additional 1:1 design for smaller viewports
+
+
+<br>
```
Comment (alisonjoseph) [inline src/content/your-product-on-ibm-cloud/widget-guidelines/usage.mdx]:
```suggestion
<br />
```

File: src/content/your-product-on-ibm-cloud/widget-guidelines/code.mdx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: Page header
```
Comment (alisonjoseph) [inline src/content/your-product-on-ibm-cloud/widget-guidelines/code.mdx]:
```suggestion
title: Widget
```

File: src/content/your-product-on-ibm-cloud/widget-guidelines/code.mdx [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+tabs: ['Usage', 'Style', 'Code']
```
Comment (alisonjoseph) [inline src/content/your-product-on-ibm-cloud/widget-guidelines/code.mdx]:
```suggestion
tabs: ['Usage', 'Style', 'Code', 'Approval']
```

**Comment by tessarodes** [comment]
Hey Alison! This is the page I intended to go there, which was part of the initial submission: https://github.com/tessarodes/carbon-website/blob/patch-1/src/content/your-product-on-ibm-cloud/widget-guidelines/review.mdx

Let me know if I messed something up and if I can help fix it!

**Pull request merged**